vespajlib/src/main/java/com/yahoo/slime/ArrayInserter.java

```diff
@@ -7,10 +7,15 @@
  **/
 public final class ArrayInserter implements Inserter {
     private Cursor target;
+
+    public ArrayInserter() {}
```

Comment:
I suggest dropping this (default ctor) and explicitly calling the other one from the decoder

Reply:
Done